airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -132,34 +139,37 @@ public void start() throws IOException {
     // anymore.
     cleanupZombies(jobPersistence, jobNotifier);
 
-    scheduleJobsPool.scheduleWithFixedDelay(
-        () -> {
-          MDC.setContextMap(mdc);
-          jobRetrier.run();
-          jobScheduler.run();
-        },
-        0L,
-        SCHEDULING_DELAY.toSeconds(),
-        TimeUnit.SECONDS);
-
-    executeJobsPool.scheduleWithFixedDelay(
-        () -> {
-          MDC.setContextMap(mdc);
-          jobSubmitter.run();
-        },
-        0L,
-        SCHEDULING_DELAY.toSeconds(),
-        TimeUnit.SECONDS);
-
-    cleanupJobsPool.scheduleWithFixedDelay(
-        () -> {
-          MDC.setContextMap(mdc);
-          jobCleaner.run();
-          jobPersistence.purgeJobHistory();
-        },
-        CLEANING_DELAY.toSeconds(),
-        CLEANING_DELAY.toSeconds(),
-        TimeUnit.SECONDS);
+    if (!featureFlags.usesNewScheduler()) {
```

Comment:
should this move to the start of the start function? we would avoid creating all these objects in memory and running the zombie kill code.

Reply:
I am concern by the first run here on an existing deployment with potential zombie sync. Do you think it can be a concern. I should probably do what you suggested, and kill the zombies in https://github.com/airbytehq/airbyte/pull/9076. Does it sounds right?

Reply:
Yeap. As is, I don't think we should rely on the old scheduler running alongside the new scheduler to deal with zombies.

Reply:
Done in the PR for the migration